### DISCUSSION TITLE
Unable to disable a workflow

### DISCUSSION THREAD

**Discussion by odysseuskir** [community/community#72826] [Actions] [answered]
### Select Topic Area

Question

### Body

Hey folks! My organization used to deploy our website using GitHub pages. We now use Vercel, and we had to disable GitHub pages. Even though I disabled GitHub pages, though, they didn't stop trying to deploy whenever we merged a new change. This has been going on for quite some time, and I can't seem to figure out how to get rid of the `pages-build-deployment` workflow. I tried [this guide](https://docs.github.com/en/actions/using-workflows/disabling-and-enabling-a-workflow?tool=webui), but I couldn't find the "Disable workflow" button. I also tried disabling it by running `gh workflow disable pages-build-deployment` on my terminal, but I got this error: `failed to disable workflow: HTTP 422: Unable to disable this workflow. (https://api.github.com/repos/simplified-coding/simplified-coding.github.io/actions/workflows/34516074/disable)` I have no idea what to do to get rid of this, and it's getting annoying, because anytime we make changes, we get annoying "Deployment failed" emails. I would appreciate any kind of help. Cheers!

**Comment by Mr-Gleidson** [comment]
It seems like you're facing issues with disabling a workflow in your GitHub repository. If you're unable to disable the workflow using the GitHub web interface or the GitHub CLI, there might be an issue that needs to be investigated. Here are some steps you can take to troubleshoot and resolve the problem:

Verify Repository Access: Ensure that you have the necessary permissions to disable workflows in the repository. You need admin or owner access to the repository to manage workflows.

Check Workflow File: Review the repository for any remaining GitHub Actions workflow configuration files (.github/workflows). If you find the workflow file that corresponds to the "pages-build-deployment" workflow, you can delete or rename the file. This will effectively disable the workflow.

Check Repository Settings:

In your GitHub repository, go to the "Settings" tab.
Navigate to the "Actions" section on the left sidebar.
Ensure that the "Actions permissions" are set to allow all actions to run.
Make sure there are no branch or event restrictions that are forcing the workflow to run.
Review GitHub Actions YAML: Open the YAML file for your workflow in the GitHub repository and check if there are any if conditions that might trigger the workflow. For example, the workflow might be triggered on specific branches or when specific events occur.

Check for External Triggers: Verify if there are any external services or webhooks that might be triggering the workflow, even if it's disabled on GitHub. Check if there are any integration services that you use for deployments which may still be configured to trigger the workflow.

GitHub Support: If the issue persists and you've exhausted all options, consider reaching out to GitHub Support. They may be able to help you resolve the issue, especially if it's related to a bug or misconfiguration.

Disable GitHub Pages: If you haven't already, ensure that GitHub Pages is disabled for the repository. To do this, go to the "Settings" of your repository, scroll down to the "GitHub Pages" section, and choose the "None" option for the source.

If, after following these steps, you are still unable to disable the "pages-build-deployment" workflow, it's recommended to reach out to GitHub support for further assistance. They can investigate the issue more deeply and provide specific guidance based on the details of your repository.

**Reply by odysseuskir** [reply]
Hey there! Thank you for your reply. I can verify that I am the owner of the repository and the organization. I tried looking up the `.github/workflows` directory but got a 404 error. I also checked the Action permissions on the repo's settings and everything looked fine. The YAML file looked fine too and there are no external triggers. As I mentioned before, I have disabled GitHub Pages. I will be contacting GitHub support, to get more info. Cheers!

**Comment by odysseuskir** [comment, marked as answer]
The solution was to rename the repository
